chromadb/utils/embedding_functions.py

```diff
@@ -115,7 +129,15 @@ def __init__(
             raise ValueError(
                 "The openai python package is not installed. Please install it with `pip install openai`"
             )
+        try:
+            import tiktoken
```

Comment:
This makes tiktoken a dependency if use openai, I don't think we want that.

Reply:
I'll just remove the checking and delegate to OAI API.